extensions/amp-video/1.0/video-wrapper.css.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const fillContentOverlay = {
+  position: 'absolute',
+  left: 0,
+  right: 0,
+  bottom: 0,
+  top: 0,
+  zIndex: 1,
```

Comment:
Is `zIndex` necessary here? We seem to run into issues with them sometimes. Does it work correctly w/o it?

Reply:
Not if we're duplicating `style`/CSS selector idents across `ContentWrapper` and `Component`, since a z-index set by author on the `VideoWrapper`. Now I'm thinking this should be a higher value, or make sure that we override `zIndex` for both of the `<Component>` and the `Autoplay` overlay layers.

Reply:
We definitely can't duplicate `style` and `className`.